maestro/chart/values.yaml

```diff
@@ -0,0 +1,130 @@
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: overture/maestro
+  tag: latest
+  pullPolicy: IfNotPresent
+
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  type: NodePort
+  port: 11235
+
+ingress:
+  enabled: false
+  annotations: {}
+  hosts:
+    - host: maestro.local
+      paths: []
+
+  tls: []
+
+resources: {}
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+appConfig:
+  server:
+    port: 80
+  maestro:
+    kafka:
+      # REPLACE this with KAFKA host/IP
+      broker: 10.11.7.31:9092
+      indexing_topic: maestro_index_requests
+    song:
+      max_retries: 3
+      timeout_sec:
+        study: 100 # some studies take really long, +30 secs, to be downloaded
+        analysis: 5
+      indexable_study_status: PUBLISHED #,UNPUBLISHED,SUPPRESSED
+    exclusion_rules:
+      # exclusion rules file name
+      file_name: exclusion-rules.yml
+
+    # elastic search server & client props
+    elasticsearch:
+      # REPLACE this with Elasticsearch host/IP
+      cluster_nodes: 10.11.7.31:9200
+      indexes:
+        file_centric:
+          alias: file_centric
+      client:
+        # this is to control the number of documents per bulk request in elasticsearch
+        docs_per_bulk_req_max: 5000
+        # max time to wait for a connection to be established
+        connection_timeout: 5000
+        # max time to wait on idle connection (no data flow)
+        socket_timeout: 10000
+        retry:
+          max_attempts: 3
+          wait_duration_millis: 500
+
+    # Genome file repositories props
+    repositories:
+      # code must be unique
+      - code: collab
+        url: https://song.cancercollaboratory.org
+        name: collaboratory
+        dataPath: /oicr.icgc/data
+        metadataPath: /oicr.icgc.meta/metadata
+        # optional
+        storageType: S3
+        organization: ICGC
+        country: CA
+      - code: aws
+        url: https://virginia.song.icgc.org
+        name: aws virginia
+        metadataPath: /oicr.icgc.meta/metadata
+        # optional
+        storageType: S3
+        organization: ICGC
+        country: US
+
+    # last resort fallback file system log in case of retries exhaustion.
+    failure_log:
+      enabled: true
+      dir: ${user.home}/logs/maestro
+
+  # logging & monitoring
+  logging:
+    level:
+      root: INFO
+      bio.overture: TRACE
+      # very verbose class, only enable lower level when necessary
+      bio.overture.maestro.domain.entities.indexing.rules.IDExclusionRule: INFO
+      org.apache.kafka.clients: WARN
+
+  # spring boot actuator endpoints
+  management:
+    endpoints:
+      web:
+        exposure:
+          include: '*'
+    endpoint:
+      health:
+        show_details: ALWAYS
+
+  spring:
+    application:
+      name: maestro
+    output.ansi.enabled: ALWAYS
+    cloud:
+      stream:
+        kafka:
+          binder:
+            brokers: ${maestro.kafka.broker}
```

Comment:
Values should be as simple as possible, it should not includes expression, it's hard to reason about.

Reply:
I understand your point, but this is to avoid repeating the value, this way it's only changed in one place when needed.
I would argue that this kind of substitution does not add complexity

Reply:
the repeated values can be avoided by defining only one variable, then spawned in the templates